fdd-no-nuget/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash
+
+set +x
+
```

Comment:
Can you add a `rm -rf ~/.nuget`?

I'm not sure it is implied by any of the `test.json` properties.

Reply:
`rm -rf ~/.nuget` is always implicit: https://github.com/redhat-developer/dotnet-bunny/blob/770c4a06f958072d935e7b88e77126a12fe469e8/Turkey/TestRunner.cs#L84

Reply:
@HoundThe can you add it anyhow. It will no-op when used with the test runner. It's still useful when the test is ran by calling `./test.sh` directly.

Reply:
I see you've added `rm nuget.config`, I meant: add a `rm -rf ~/.nuget` line.

Reply:
Oh, lol. Sorry, fixed it.